domain/service_broker.go

```diff
@@ -169,6 +170,7 @@ type UnbindSpec struct {
 
 type Binding struct {
 	IsAsync         bool          `json:"is_async"`
+	AlreadyExists   bool          `json:"-"`
```

Comment:
Curious why this `json:"-"` annotation rather than `json:"already_exists"`? cc @FelisiaM